app/scripts/controllers/transactions/index.js

```diff
@@ -25,6 +25,11 @@ const {
   TRANSACTION_TYPE_RETRY,
 } = require('./enums')
 
+const ContractKit = require('@celo/contractkit')
```

Comment:
Mixing commonjs require and es6 import here

Reply:
Thanks, James! I replaced `require`s with ES6 imports statements in https://github.com/poanetwork/nifty-wallet/pull/446/commits/86a08d052f8017804cf903183df0891219be654a specifically for TransactionController. Though I am aware, there can be multiple places where mixing takes places in the code. I suppose other mixing occurrences can be processed in a separate refactoring PR(s).